src/app/course/components/btn-load-more/btn-load-more.component.ts

```diff
@@ -26,7 +25,7 @@ export class CoursesLoadMoreComponent implements OnInit {
 	onLoadMore() {
 		this.showLoader = true;
 		this.coursesService.getList().then((items: ICourse[]) => {
-			map(items, (item: CourseProps) => this.listCourses.push(new Course(item)));
+			map(items, (item: TCourse) => this.listCourses.push(new Course(item)));
```

Comment:
Do not use map for simple iterating, please. It's a bad practice.

Reply:
fixed using map